src/epcpm/staticmodbustoc.py

```diff
@@ -216,13 +218,18 @@ def gen(self) -> typing.List[str]:
         Returns:
             list: staticmodbusAddrRegMap rows for the generated .c file output
         """
-        # TODO: to be implemented, for now NULL for all FunctionDataBitfield objects
+        # TODO: CAMPid 9685439641536675431653179671436
```

Comment:
How about a helper.py file that has some of these repeated functions?

Reply:
I added `pm_helper.py` and refactored `CAMPid 9685439641536675431653179671436` and `CAMPid 07954360685417610543064316843160`. (Note: both of these CAMPid's turned out to be the same.) In addition, there were spots that were not marked with either CAMPid that also needed to be refactored. Fixed in 6091c44c4e0c921e0eaa944b9ddfff75e11dcdd4

I'm not going to do any more of these CAMPid refactors in this PR as each one is a rabbit hole. I recommend they are addressed in a future PR (or PR's).

Reply:
Totally agree